samples/HostedInAspNet.Client/wwwroot/index.html

```diff
@@ -7,6 +7,6 @@
 <body>
     <app>Loading...</app>
     <script src="customJsFileForTests.js"></script>
-    <script type="blazor-boot"></script>
+    <script src="_framework/blazor.server.js"></script>
```

Comment:
To keep the WebAssembly version consistent with the server-executed version, I'm thinking we should change the WebAssembly startup mechanism to use a tag like:

    <script src="_framework/blazor.webassembly.js"></script>

... instead of the magic `<script type="blazor-boot">`. It would work by first issuing a request for a `_framework/appconfig.json` file that was generated during the build and contains the same information that we normally put onto the <script type="blazor-boot">.

This also has the benefit that we no longer need to "build" the `index.html` file; it just becomes a normal static file. In turn this means the developer can either serve the `index.html` from their client app or put the `<script>` tag in any server-rendered page.

@rynowak If you agree, I'll implement that in a separate PR.

Reply:
Yeah, this sounds great.  I'd suggest something like `_framework/blazorconfig.json` since appconfig is a common enough name (yes I realize its a separate directory).

Reply:
Great, I'll do that.